Divorce triton and pt2 remote caching

oulgen opened the pull request:
Now that remote caching has evolved into various parts of PT2, we want to separate triton and pt2 caching as changes to one have caused SEVs to the other.


Differential Revision: D60047752




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

aorenste reviewed (approved):
Needs a better summary.

oulgen commented:
@aorenste yup, i just exported from fbcode, will add

oulgen committed:
Divorce triton and pt2 remote caching (#131345)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/131345

Now that remote caching has evolved into various parts of PT2, we want to separate triton and pt2 caching as changes to one have caused SEVs to the other.

Test Plan: Existing uint tests and ad hoc testing

Reviewed By: c00w, aorenste

Differential Revision: D60047752
